front/src/common/Map/Search/MapSearchStation.tsx

```diff
@@ -74,7 +74,7 @@ const MapSearchStation = ({ updateExtViewport, closeMapSearchPopUp }: MapSearchS
           setNameResults([]);
           setTrigramResults(results as SearchResultItemOperationalPoint[]);
         }
-        if (isSearchingByName) {
+        if (!isSearchingByTrigram) {
```

Comment:
I think you can use a else here
```suggestion
       else {
```